clang/include/clang/StaticAnalyzer/Core/PathSensitive/SVals.h

```diff
@@ -514,9 +514,12 @@ class MemRegionVal : public Loc {
 
 class ConcreteInt : public Loc {
 public:
-  explicit ConcreteInt(const llvm::APSInt &V) : Loc(ConcreteIntKind, &V) {}
+  explicit ConcreteInt(APSIntPtr V) : Loc(ConcreteIntKind, V.get()) {}
```

Comment:
Is it possible to store `APSIntPtr` in `Loc`?

Reply:
In short, I don't think. See https://github.com/llvm/llvm-project/pull/120436#discussion_r1890504894.

Reply:
I guess this one is resolved now. Is it?
Anything else blocking this PR?